.eslintrc.js

```diff
@@ -44,7 +44,7 @@ module.exports = {
 		'no-unreachable': 1,
 		// Allows Chai `expect` expressions
 		'no-unused-expressions': 0,
-		'no-unused-vars': 1,
+		'no-unused-vars': 2,
```

Comment:
Because we extend [`eslint-config-wpcalypso`](https://github.com/Automattic/eslint-config-wpcalypso) we can simply remove the line here because it only existed to downgrade from the default error to warning.